packages/vue/src/components/organisms/SfTabs/SfTabs.vue

```diff
@@ -38,17 +38,24 @@ export default {
       initialTabActivated: false,
     };
   },
+  watch: {
+    openTab(newValue, oldValue) {
+      if (newValue === oldValue) return;
+      this.toggle(this.$children[newValue - 1]._uid);
+    },
+  },
   mounted() {
     this.$on("toggle", this.toggle);
     if (this.openTab) this.openChild();
   },
   methods: {
     toggle(id) {
       this.$children.forEach((child) => {
-        child._uid === id
-          ? (child.isActive = !child.isActive)
-          : (child.isActive = false);
+        child._uid === id ? (child.isActive = true) : (child.isActive = false);
       });
+      const active =
```

Comment:
Maybe change name of the variable to something connected to index? I.e
```suggestion
      const activeTabId =
```